docs/tutorial_kp.md

```diff
@@ -20,17 +20,15 @@ build an OCI image from source and allow kpack rebuild the OCI image with update
 
     ```bash
     docker login <REGISTRY-HOSTNAME> -u <REGISTRY-USER>
-    kp secret save kp-default-registry-creds \
-       --registry <REGISTRY-HOSTNAME> \
-       --registry-user <REGISTRY-USER> \
-       -n kpack
+    kp secret create kp-default-registry-creds \
```

Comment:
by default this should probably still be `--registry` instead of `--gcr` because we shouldn't assume everyone is using gcr.